train/run.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Do we want to make this public?

Reply:
yeah. they do not have private info and that is needed to run.